packages/kbn-management/settings/components/field_input/types.ts

```diff
@@ -31,7 +31,9 @@ export interface FieldInputServices {
  */
 export interface FieldInputKibanaDependencies {
   /** The portion of the {@link ToastsStart} contract used by this component. */
-  toasts: Pick<ToastsStart, 'addDanger'>;
+  notifications: {
+    toasts: Pick<ToastsStart, 'addDanger'>;
```

Comment:
Again, restricting the `notifications` property of `CoreStart` to not only the specific namespace we want-- `toasts`-- but also the specific methods we use.